_includes/schedule.html

```diff
@@ -48,7 +48,7 @@ <h5 class="slot-title" itemprop="name">{{ session.title }}</h5>
                                                     {% if session_speaker == speaker.id %}
                                                     <li itemprop="performer"><div class="speaker-img flow-img img-circle" style="background-image: url({{ site.baseurl | append: '/img/people/' | append: speaker.thumbnailUrl }})"></div>
                                                     <p class="speaker-name">{{ speaker.name }} {{ speaker.surname }}
-                                                        <span class="speaker-position">{{ speaker.company }}</span>
+                                                        <span class="speaker-position">{{ speaker.company[0].title }}, {{ speaker.company[0].name }}</span>
```

Comment:
@inovizz Can you use the format I have used [here](https://github.com/HydPy/hydpyconf2017/commit/bc25d886cb8ea52e0a431f5869084c9606ad049e#diff-c0c978b7464e8306eb9b5d12268e9db9R24). We should provide all the company the Speaker works with.